benchmarks/src/main/scala/monix/connect/benchmarks/redis/laserdisc/RedisLaserdiscHashesBenchmark.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2020-2020 by The Monix Connect Project Developers.
+ * See the project homepage at: https://connect.monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.connect.benchmarks.redis.laserdisc
+
+import cats.implicits._
+import laserdisc.fs2._
+import laserdisc.{Key, all => cmd}
+import org.openjdk.jmh.annotations._
+
+import scala.concurrent.Await
+import scala.concurrent.duration.DurationInt
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@Measurement(iterations = 5)
+@Warmup(iterations = 1)
+@Fork(1)
+class RedisLaserdiscHashesBenchmark extends RedisLaserdiscBenchFixture {
```

Comment:
Would it be possible to group benchmarks by "scenario", rather than "library"?

e.g. `RedisHashesBenchmark` would measure hash operations for all libraries in one class. Like these benchmarks: https://github.com/zio/zio-redis/blob/master/benchmarks/src/main/scala/zio/redis/PutBenchmarks.scala

We could either choose one library to do setup, or have write + read in the same scenario instead of separate ones.

I feel like it would be a more consistent setup and comparisons would be a little bit more fair

Reply:
Yes, seems that it would be better. I'll make it that way